ngfizzy/ps-notes
# add functionality to sighup UI(auth):

### WHAT DOES THIS PR DO?
This PR implemented signup functionality.

### HOW CAN THIS BE MANUALLY TESTED

* Go to localhost:4000
* Click on signup button
* Provide Necessary details
* Click on signup
* You should see an alert message that indicates that your signup was successful or 
* An error message if there was an error during signup
